extensions/tailscale/src/exit.tsx

```diff
@@ -1,16 +1,24 @@
-import { ActionPanel, List, Action, popToRoot, closeMainWindow, Image, Icon } from "@raycast/api";
+import { Action, ActionPanel, closeMainWindow, Icon, Image, List, popToRoot } from "@raycast/api";
 import { useEffect, useState } from "react";
 import {
-  StatusResponse,
-  getStatus,
-  getDevices,
-  tailscale,
-  sortDevices,
-  ErrorDetails,
-  getErrorDetails,
-  Device,
+    Device,
+    ErrorDetails,
+    getDevices,
+    getErrorDetails,
+    getStatus,
+    sortDevices,
+    StatusResponse,
+    tailscale,
 } from "./shared";
 
+function getCountryFlag(name: string): string {
+  const countryCode = name.substring(0, 2).toUpperCase();
+  if (countryCode.match(/^[A-Z]{2}$/)) {
+    return String.fromCodePoint(...[...countryCode].map(c => 0x1F1A5 + c.charCodeAt(0)));
```

Comment:
What happens if the device name starts with any country code combination, even if it's not meant to be a country code?

And second, this makes the list fully misaligned, because not all rows will have the initial emoji. For this a solution would be to add it to the subtitle or the other side of the row.